internal/pkg/agent/application/monitoring/process.go

```diff
@@ -33,6 +53,18 @@ func processHandler(coord *coordinator.Coordinator, statsHandler func(http.Respo
 			return statsHandler(w, r)
 		}
 
+		metricsPath := vars["metricsPath"]
```

Comment:
[Suggestion]
As `processID` is a const, make it one as well. Personally I'd say all the vars names should be constants to avoid typos or bug if they ever change